### DISCUSSION TITLE
change name of mqtt payload

### DISCUSSION THREAD

**Discussion by jarvissd** [arendst/Tasmota#20935] [Q&A]
### PROBLEM DESCRIPTION

hi, im a newbie . i had a problem about changing name of payload . For ex, when i publish "on" to turn on led power with topic "cmnd/tasmota_01/Power" , i will receive a payload message: <Power: on> from MQTT . here, i can't change payload message like <led1:on> instead of <Power: on> . i tried to use sonof.ino but it didn't work 

### REQUESTED INFORMATION
_Make sure your have performed every step and checked the applicable boxes before submitting your issue. Thank you!_

- [x] Read the [Contributing Guide and Policy](https://github.com/arendst/Tasmota/blob/development/CONTRIBUTING.md) and [the Code of Conduct](https://github.com/arendst/Tasmota/blob/development/CODE_OF_CONDUCT.md)
- [x] Searched the problem in [issues](https://github.com/arendst/Tasmota/issues)
- [x] Searched the problem in [discussions](https://github.com/arendst/Tasmota/discussions)
- [x] Searched the problem in the [docs](https://tasmota.github.io/docs/FAQ)
- [x] Searched the problem in the [chat](https://discord.gg/Ks2Kzd4)
- [x] Device used (e.g., Sonoff Basic): _____
- [x] Tasmota binary firmware version number used: _____
  - [x] Pre-compiled
  - [x] Self-compiled
- [x] Flashing tools used: _____
- [ ] Provide the output of command: `Backlog Template; Module; GPIO 255`:
```lua
  Configuration output here:


```
- [ ] If using rules, provide the output of this command: `Backlog Rule1; Rule2; Rule3`:
```lua
  Rules output here:


```
- [ ] Provide the output of this command: `Status 0`:
```lua
  STATUS 0 output here:


```
- [ ] Set `weblog` to 4 and then, when you experience your issue, provide the output of the Console log:
```lua
  Console output here:


```

### TO REPRODUCE
_Steps to reproduce the behavior:_

### EXPECTED BEHAVIOUR
_A clear and concise description of what you expected to happen._

### SCREENSHOTS
_If applicable, add screenshots to help explain your problem._

### ADDITIONAL CONTEXT
_Add any other context about the problem here._

**(Please, remember to close the issue when the problem has been addressed)**

**Comment by Jason2866** [comment]
Read the Tasmota Documentation. The payload is not designed to be changed by default.

**Reply by jarvissd** [reply]
this means it can't be changed. Right? 
thanks a lot

**Reply by sfromis** [reply]
What you *can* do instead is to use [rules](https://tasmota.github.io/docs/Rules/) to publish additional payloads in your own format. Takes a learning curve to get started with that.
